src/app/Http/Controllers/DuplicateController.php

```diff
@@ -136,4 +137,30 @@ public function resolve(Duplicate $duplicate) {
         $duplicate->save();
         return redirect()->route('duplicate.index');
     }
+
+    /**
+     * This function is used to mark multiple duplicates as resolved
+     *
+     * @param Request $request
+     * @return RedirectResponse
+     */
+    public function multiple_resolve(Request $request) {
+        $input = $request->all();
```

Comment:
Use a personalized request file instead of a Global request

/app/http/Requests